force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -281,7 +336,7 @@ public with sharing class ClassesObjectsMethods {
 
         // Add the book to the list
         
-        return null; // Replace null with the variable you used to store the result      
+        return myListBooks; // Replace null with the variable you used to store the result      
```

Comment:
excellent. Something you can do is to assign to the list directly.

 books.add(new Book('A Brief History of Time', 'Stephen Hawking', 256));